x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/action/TransportStartTrainedModelDeploymentAction.java

```diff
@@ -114,18 +119,33 @@ protected void masterOperation(Task task, StartTrainedModelDeploymentAction.Requ
                         request.getModelId(), getModelResponse.getResources().results().size()));
                     return;
                 }
+
+
+                TrainedModelConfig trainedModelConfig = getModelResponse.getResources().results().get(0);
+                if (trainedModelConfig.getModelType() != TrainedModelType.PYTORCH) {
```

Comment:
I am assuming that this is just for this development spike. Before merging we will allow deploying of all models (I would think)

Reply:
I thought it was a sane check to perform right now but may not make sense in the future